spec/Section 6 -- Execution.md

```diff
@@ -322,39 +312,290 @@ Unsubscribe(responseStream):
 
 - Cancel {responseStream}.
 
-## Executing Selection Sets
+## Executing the Root Selection Set
+
+To execute the root selection set, the object value being evaluated and the
+object type need to be known, as well as whether it must be executed serially,
+or may be executed in parallel.
+
+Executing the root selection set works similarly for queries (parallel),
+mutations (serial), and subscriptions (where it is executed for each event in
+the underlying Source Stream).
+
+First, the selection set is turned into a grouped field set; then, we execute
+this grouped field set and return the resulting {data} and {errors}.
+
+ExecuteRootSelectionSet(variableValues, initialValue, objectType, selectionSet,
+serial):
+
+- If {serial} is not provided, initialize it to {false}.
+- Let {groupedFieldSet} and {newDeferUsages} be the result of
+  {CollectFields(objectType, selectionSet, variableValues)}.
+- Let {executionPlan} be the result of {BuildExecutionPlan(groupedFieldSet)}.
+- Let {data} and {incrementalDataRecords} be the result of
+  {ExecuteExecutionPlan(newDeferUsages, executionPlan, objectType, initialValue,
+  variableValues, serial)}.
+- Let {errors} be the list of all _field error_ raised while completing {data}.
```

Comment:
```suggestion
- Let {errors} be the list of all _field error_ raised while completing {data}.
+ Let {errors} be the list of all _field error_ raised while executing the execution plan.
```
This might be more clear? and its similar to the previous language used in ExecuteQuery, etc

Reply:
0058d2a

The note is a WIP, feel free to further edit